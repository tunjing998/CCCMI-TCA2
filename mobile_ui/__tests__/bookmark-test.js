import React, {Component} from 'react';
import {View, Text} from 'react-native';
import BookmarkScreen from '../screens/BookmarkScreen';
import renderer from 'react-test-renderer';

test('Splash screen snapShot', () => {
  const snap = renderer.create(<BookmarkScreen />).toJSON();
  expect(snap).toMatchSnapshot();
});
