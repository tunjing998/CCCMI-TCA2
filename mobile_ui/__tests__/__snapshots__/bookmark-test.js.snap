// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Splash screen snapShot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Bookmark Screen
  </Text>
  <View
    accessibilityRole="button"
    accessibilityStates={Array []}
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Click Here
      </Text>
    </View>
  </View>
</View>
`;
